docs/modules/syntax-highlighting/pages/pygments.adoc

```diff
@@ -8,16 +8,14 @@
 
 == Install Pygments
 
-In order to use Pygments with Asciidoctor, you need {url-python}[Python 2^] and the {url-pygments-gem}[pygments.rb gem^].
+In order to use Pygments with Asciidoctor, you need {url-python}[Python 3^] and the {url-pygments-gem}[pygments.rb gem^].
 The pygments.rb gem manages calls to Pygments, which is an external program that runs using Python.
 
 TIP: You do not need to install Pygments itself.
 It comes bundled with the pygments.rb gem.
 
-IMPORTANT: You must have Python 2 installed to use pygments.rb.
-Python 3 is not compatible with the pygments.rb gem.
-Check that you have a `python2` (Linux), `python` (macOS), or `py -2` (Windows) executable on your PATH.
-(On macOS, verify that the `python` executable uses Python 2 by running `python -V`).
+IMPORTANT: You must have Python 3 installed to use pygments.rb.
```

Comment:
I think we should clarify here that you can have either Python 2 or Python 3. But if you must use Python 2, then you have to use the older (unsupported version) of pygments.rb. Of course we are discouraging it, but the information should help those who are trying to migrate.

Reply:
All this page needs a major rework. It tells users to do scary things to workaround the fact that pygments.rb cannot use non-bundled Pygments version currently.

Reply:
We're not doing a general review of the docs right now (and beside, most of this will be removed now that docs.asciidoctor.org is online anyway). So the revisions happen later. For now, we just cannot make assertions that aren't entirely true. Python 3 is not all of a sudden required. It is required if you use pygments.rb 2. If you use pygments.rb 1, then Python 2 is required. That's all I'm asking.

Reply:
Sure, no problem. Besides that, tests need to be fixed before this can proceed anywhere :)